Directory.Packages.props

```diff
@@ -12,7 +12,7 @@
   <ItemGroup>
     <!-- Azure SDK for .NET dependencies -->
     <PackageVersion Include="Azure.AI.Inference" Version="1.0.0-beta.5" />
-    <PackageVersion Include="Azure.AI.OpenAI" Version="2.2.0-beta.5" />
+    <PackageVersion Include="Azure.AI.OpenAI" Version="2.3.0-beta.1" />
```

Comment:
Won't this break until Aspire can also reference M.E.AI.Abstractions 9.8?

Reply:
The breaking binary change is between Aspire and Open AI - so long as both of those are upgraded I think it will be fine.

I didn't notice any binary breaking changes you reacted to in ME.AI -- let me check.